mesonbuild/mesonmain.py

```diff
@@ -26,10 +26,10 @@
 import importlib
 import argparse
 
-from .utils.core import MesonException, MesonBugException
+from .utils.core import MesonException, MesonBugException, MesonExceptionWrapper
 from . import mlog
 
-def errorhandler(e, command):
+def errorhandler(e: Exception, command: str) -> int:
     import traceback
     if isinstance(e, MesonException):
```

Comment:
I wonder if we should remove the `else` below and unindent the rest of the function...

Reply:
Is it actually important to shift over big blocks of code just because an else is "useless"? Why is it useless?

It could be argued it helps you visually eyeball the logic flow. That's the reason I originally wrote the code that way... I suppose opinions may vary.

Is it more efficient python bytecode? Do you feel it simply looks prettier?

Reply:
imho, reducing indentation level is often a good thing. It helps reduccing Christmas tree code...

Reply:
I don't think this particular code had that problem. It's a single indent level for all child code being dedented here.

"Christmas tree code" tends to have unmistakable tells.